docs/reference/method/MongoDBCollection-createSearchIndexes.txt

```diff
@@ -91,18 +91,23 @@ to index all document fields containing
 
    $collection = (new MongoDB\Client)->selectCollection('test', 'articles');
 
-   $indexName = $collection->createSearchIndex(
-      ['mappings' => ['dynamic' => true]],
-      ['name' => 'test-search-index']
-   );
+   $indexNames = $collection->createSearchIndexes([
```

Comment:
You can put this first square bracket on a new line to make it more evident that it's a list of indexes.

Reply:
This was consistent with what we already had in `createIndexes()` but I won't object.

Reply:
Perfect